web/.vscode/settings.json

```diff
@@ -5,5 +5,5 @@
   },
   "prettier.eslintIntegration": true,
   "jest.pathToConfig": "jest.config.js",
-  "jest.restartJestOnSnapshotUpdate": true
+  "local-history.path": "C:\\Projects\\Integreat\\intregreat-app backup"
```

Comment:
```suggestion
  "jest.restartJestOnSnapshotUpdate": true
```

Reply:
Maybe we should add .vscode/settings.json to gitignore as it also contains user specific settings?

Reply:
Sure :+1: